modules/gke-cluster-standard/main.tf

```diff
@@ -31,6 +31,7 @@ resource "google_container_cluster" "cluster" {
   enable_intranode_visibility = var.enable_features.intranode_visibility
   enable_l4_ilb_subsetting    = var.enable_features.l4_ilb_subsetting
   enable_shielded_nodes       = var.enable_features.shielded_nodes
+  enable_fqdn_network_policy  = var.enable_features.dataplane_v2 && var.enable_features.fqdn_network_policy ? true : false
```

Comment:
I'd simply pass the value without checking if if dataplane_v2 is set. Let the API fail if what the user specifies is inconsistent.

Alternatively add a validation directly in the variable.

Reply:
Done :)